docs/guide/styles.md

```
@@ -70,10 +70,93 @@ defstyle style-input $ {}
+```cirru
+div $ {}
+  :class-name $ str-spaced css/row style-preview-row
+```
+
+```cirru
+defstyle style-preview-row $ {}
+  |& $ {} (:gap |8px) (:flex-wrap :wrap)
+    :align-items :flex-start
+```
+
+Be careful with string values like `|4px 10px`, `|1px solid `, or long text literals. If the code is generated or transformed by CLI tools, prefer the exact serialized form that the tool prints, instead of hand-editing token boundaries.
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)